src/Resolver/DefaultValueResolver.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neusta\ConverterBundle\Resolver;
+
+final class DefaultValueResolver
+{
+    public static function resolve(array $config)
+    {
+        return $config['default'] ?? null;
+    }
+}
```

Comment:
Do we really need this class? I think the code should just be inlined in the extension class.

Reply:
Not yet, but I've been considering implementing a feature that allows the addition of a default value to a specific property